src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -736,7 +741,7 @@ DetailsCard.propTypes = {
 }
 
 const DetailsCardConnected = connect(
-  (state) => ({
+  (state, { vm }) => ({
```

Comment:
Why do you have `{ vm }` here?

Reply:
It's a hold over from merging.  This is what it looked like in just the permissions #784:
https://github.com/oVirt/ovirt-web-ui/blob/b05b015d1c1cd1bfb6ab3df7b03742f73fdc1cd1/src/components/VmDetails/cards/DetailsCard.js#L42-L47

No need for it now, I'll take it out.